### PR TITLE
oAuth: add github and google login option

### LINKED ISSUE #126: Add login
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [ X] feature request
- [ ] support request

**Expected behavior:**
Add login for a user

### PR DESCRIPTION
**Pull request by sagar-kalra**
Added option for users to login using their github and google account.

Fixes #126

### CODE REVIEW

File: pclubWebsite/settings.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 SECRET_KEY = '$4p#tsin((&rnm!%696^2xutz_-n5k)o&mnwg3z*^f7&--99h&'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = True
```
Comment (gurpreetsingh00885) [inline pclubWebsite/settings.py]: You need to revert this change.

Comment (gurpreetsingh00885) [reply pclubWebsite/settings.py]: Otherwise LGTM. Just change the redirect uri.
We can have new issues to make the templates look prettier.

**Comment by divyam3897** [comment]
@gurpreetsingh00885 can you deploy the PR?

**Comment by gurpreetsingh00885** [comment]
plus add pclubuiet-pr-153.herokuapp.com/... as a redirect uri
